### ISSUE TITLE
Update X logo to 𝕏 on the website's footer 

### ISSUE THREAD

**Issue by ArvindParekh** [kubernetes/contributor-site#503]
In the footer, we still have the old bird icon for Twitter. I suggest we update that to 𝕏.

![image](https://github.com/kubernetes/contributor-site/assets/71211731/4cf8a569-3389-4c3a-9014-b87ebfd1367e)

**Comment by jberkus** [comment]
That's correct.  However, if I'm the one to fix it, it's going to wait until I can add the icons for all our social media accounts.

**Comment by ArvindParekh** [comment]
Yes. Probably an update to the Docsy version is required to use the latest fontawesome icons (which would also impact the entire site). 

Here's a similar issue, for the [kubernetes.io](https://kubernetes.io) site: https://github.com/kubernetes/website/issues/46405

**Comment by ArvindParekh** [comment]
@SayakMukhopadhyay so we'll be able to add it once the site has been updated to Docsy 0.9.0, correct?

**Comment by SayakMukhopadhyay** [comment]
> @SayakMukhopadhyay so we'll be able to add it once the site has been updated to Docsy 0.9.0, correct?

@ArvindParekh Yeah, I will work on the futher upgrades and I will incorporate this fix in my PR which upgrades to 0.9.0 or later. It would be great if you can reopen it so that its tracked.

**Comment by ArvindParekh** [comment]
Will do, thanks!
/reopen
/priority backlog
/kind documentation

**Comment by jberkus** [comment]
FYI, once we have the upgrade, I'm going to submit a PR to add all our socials to the footer.

**Comment by SayakMukhopadhyay** [comment]
This issue is still valid
/reopen

**Comment by Aaina26** [comment]
/remove-lifecycle rotten

As this issue still need to be resolved

**Comment by SayakMukhopadhyay** [comment]
Hi @richochetclementine1315 the core issue is that the bird logo of twitter needs to be changed to the X logo of, well, X. But the Fontawesome version that is currently bundled with Docsy is a bit old and doesn't have the new X logo. So, for this to be fixed, we need to migrate to Docsy version 0.9.0, which has a fontawesome version that contains the X logo.

**Comment by jberkus** [comment]
We also need the new FA in order to add icons for other social media sites, like Bluesky and Linkedin.

**Comment by SayakMukhopadhyay** [comment]
/remove-lifecycle rotten

This issue is still valid and is dependant on the Docsy upgrade.
